### ISSUE TITLE
Quick notes on updating and deploying the website

### ISSUE THREAD

**Issue by tompollard** [acm-chil/acm-chil-website#16]
Here are some quick notes on how to make changes to the website (@tnaumann @alistairewj @corcra @irenetrampoline @mmcdermott):

1) Install the requirements with `pip install requirements.txt` (or similar) preferably in a virtual environment.
2) Check out a new working branch with `git checkout -b NAME_OF_BRANCH`. Make your changes!
3) Test your changes locally by running `make run`. you should be able to view the site at: http://localhost:5000/
4) If you are happy, `git push` your branch to github and open a pull request to the main branch.
5) Once a change is merged, run `make deploy` to deploy the changes to the live website.

If you see a new pull request, feel free to check it and merge it yourself! I think it will be easier to keep on top of things if there are a few people keeping an eye out for changes.

(When you deploy, you'll see some "Filename extension" warnings, which can be ignored. It looks like these relate to the `.html` extensions that are used in the template. These extensions aren't needed, but removing them now will break links that we've shared).

**Comment by tnaumann** [comment]
N.B. before running `make deploy` you will likely need to create the two branches it's expecting to find as part of `git branch -D`:
```
git branch gh-pages
git branch temp-gh-pages
```

**Comment by tompollard** [comment]
@tnaumann This should fixed since https://github.com/acm-chil/acm-chil-website/commit/b4502f5cd26f5940aad9ef27798fc264aa242856 (the hyphens at the start of the line allow the build to continue even if the branches don't exist).
